traffic_ops/install/bin/_postinstall

```diff
@@ -28,7 +28,7 @@ use DBI;
 use POSIX;
 use File::Basename qw{dirname};
 use File::Path qw{make_path};
-use Digest::SHA1 qw(sha1_hex);
+use Crypt::ScryptKDF qw(scrypt_hash);
```

Comment:
doesn't appear to be in there..   please add to that module to `traffic_ops/app/cpanfile`